lib/environment.php

```diff
@@ -102,7 +102,7 @@ function merge_config_files($folder_path) {
      * @return array The configuration array.
      */
     function config($file_name) {
-        $path = CONFIG_PATH.$file_name.'.php';
+        $path = APP_PATH.'config/'.$file_name.'.php';
```

Comment:
CONFIG_PATH is defined in index.php, why don't you use it?

Reply:
Copy that, I'l have to redeclare it also in tests/phpunit/bootstrap.php to use it withtout error.